moto/cloudformation/parsing.py

```diff
@@ -504,11 +504,26 @@ def transform_mapping(self):
                     key = s3_backend.get_object(bucket_name, name)
                     self._parsed_resources.update(json.loads(key.value))
 
+    def parse_ssm_parameter(self, value, value_type):
+
+        # The Value in SSM parameters is the SSM parameter path
+        # we need to use ssm_backend to retreive the
+        # actual value from parameter store
+        parameter = ssm_backends[self._region_name].get_parameter(value, False)
+        actual_value = parameter.value
+        if value_type.find("List") > 0:
+            return actual_value.split(",")
+        return actual_value
+
     def load_parameters(self):
         parameter_slots = self._template.get("Parameters", {})
         for parameter_name, parameter in parameter_slots.items():
             # Set the default values.
-            self.resolved_parameters[parameter_name] = parameter.get("Default")
+            value = parameter.get("Default")
+            value_type = parameter.get("Type")
+            if parameter.get("Type").startswith("AWS::SSM::Parameter::") and value:
```

Comment:
`parameter.get("Type")` was already resolved, so this can be simplified to `if value_type.startswith...`

Reply:
@bblommers Absolutely. Fixed: e9585a2